pkgs/applications/kde/kontact.nix

```diff
@@ -19,5 +21,12 @@ mkDerivation {
     kcmutils kcrash kdbusaddons kparts kwindowsystem
     akonadi grantleetheme kontactinterface kpimtextedit
     mailcommon libkdepim pimcommon
-  ];
+  ]
+  ++ pimExternalApplications.kmailApplications;
+  # create wrapper for being able to call external executables,
+  # please concatenate the external dependencies of all PIM components specified in `pimExternalApplications`
+  postFixup = ''
+    wrapProgram "$out/bin/kontact" \
+    --prefix PATH : "${lib.makeBinPath ([] ++ pimExternalApplications.kmailApplications)}"
```

Comment:
```suggestion
    --prefix PATH : "${lib.makeBinPath pimExternalApplications.kmailApplications}"
```

Reply:
This is actually deliberate and I'd like to keep it the way it is:
Currently we only know of KMail requiring external dependencies, but there might be some in other parts of the PIM suite as well we're still missing (KOrganizer, Akregator, …).  This concatenation is supposed to make it clear to future PIM maintainers that dependencies of different PIM parts need to be concatenated.